action-templates/actions/action-build-docs/action.yml

```diff
@@ -1,27 +1,23 @@
 # https://vitepress.dev/guide/deploy#github-pages
-name: Build docs
+name: "Build docs"
```

Comment:
also hier macht @ejcsid wieder alles Rückgängig https://github.com/it-at-m/lhm_actions/pull/68/files#r2043683520

Reply:
Ah da hatte ich den anderen PR übersehen sorry.
Mir egal wie rum wirs machen, kann hier auch bei name und description der action die Anführungszeichen auch wieder entfernen

Reply:
im verlinkten PR ist description mit ", name ohne. ist aber besser es einheitlich zu machen. Mein PR kann im wegen Breaking Changes nicht gemerged werden. Dann übernehme ich es wie es hier ist (überall mit Hochkommas)